include/deal.II/distributed/cell_data_transfer.h

```diff
@@ -302,8 +302,8 @@ namespace parallel
           cell_iterator &cell,
         const CellStatus status,
         const boost::iterator_range<std::vector<char>::const_iterator>
-                                  &data_range,
-        std::vector<VectorType *> &all_out);
+                                        &data_range,
+        const std::vector<VectorType *> &all_out);
```

Comment:
I think all of the others are reasonable, but this is going to confuse people. The point is that it's a vector to writable things. It's try that the vector itself is const, it's just that it stores pointers to non-const.

Reply:
Sure.